manifests/charts/gateways/istio-egress/templates/deployment.yaml

```
@@ -263,7 +263,11 @@ spec:
 {{- if eq .Values.global.pilotCertProvider "istiod" }}
       - name: istiod-ca-cert
         configMap:
+{{- if and (hasKey .Values "experimental") .Values.experimental.caRootCertConfigMap }} 
```

```suggestion
{{- if and (hasKey .Values "experimental") .Values.experimental.caRootCertConfigMap }} 
```

I kind of object to introducing an `experimental` group, I think it adds nothing but complication to the UX and another weird Helm convention we have to explain to people.

What's the point of doing `.Values.experimental.foo` versus just `.Values.foo` in a template? The latter will work just as well in all usages I see here.

What's experimental about it? We either template it if you set it, or do not if you do not.